pkg/cri/server/service.go

```diff
@@ -417,6 +417,9 @@ func loadBaseOCISpecs(config *criconfig.Config) (map[string]*oci.Spec, error) {
 			return nil, fmt.Errorf("failed to load base OCI spec from file: %s: %w", cfg.BaseRuntimeSpec, err)
 		}
 
+		if spec.Process != nil && spec.Process.Capabilities != nil && len(spec.Process.Capabilities.Inheritable) > 0 {
+			log.L.Warn("Inheritable capabilities were removed from the default spec.")
```

Comment:
I don't think this message is very clear.  Maybe something like this:

```suggestion
			log.L.WithField("base_runtime_spec", cfg.BaseRuntimeSpec).Warn("Provided base runtime spec includes inheritable capabilities, which may be unsafe. See CVE-2022-24769 for more details.")
```

Reply:
Yes, if we provide the CVE ID, the reason for the warning can be explained more clearly.